src/vm/exceptionhandling.cpp

```diff
@@ -362,14 +362,26 @@ void ExceptionTracker::UpdateNonvolatileRegisters(CONTEXT *pContextRecord, REGDI
         pAbortContext = GetThread()->GetAbortContext();
     }
 
```

Comment:
Yes, I do. It was my testing app for exception rethrow from the catch. Sergiy's original exception testing app would do the same since it happens before the rethrow.